mcp/1.1/modules/ROOT/pages/mcp-connector-reference.adoc

```diff
@@ -0,0 +1,1120 @@
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Criticality a| Boolean |  |  | 
+| Oid a| String |  |  | 
+| Value a| Binary |  |  | 
+|===
+
+[[AlternativeNameData]]
+=== Alternative Name Data
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Name a| String |  |  | 
+| Type a| Number |  |  | 
+|===
+
+[[PrincipalData]]
+=== Principal Data
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Common Name a| String |  |  | 
+| Name a| String |  |  | 
+|===
+
+[[X500PrincipalData]]
+=== X500 Principal Data
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Name a| String |  |  | 
+|===
+
+[[PublicKeyData]]
+=== Public Key Data
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Algorithm a| String |  |  | 
+| Encoded a| Binary |  |  | 
+| Modulus a| Number |  |  | 
+| Params a| Number |  |  | 
+| Public Key a| String |  |  | 
+|===
+
+[[SerialNumberData]]
+=== Serial Number Data
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Serial Number a| Number |  |  | 
+|===
+[[RedeliveryPolicy]]
+=== Redelivery Policy
+
+Configures the redelivery policy for executing requests that generate errors. You can add a redelivery policy to any source in a flow.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Redelivery Count a| Number | Maximum number of times that a redelivered request can be processed unsuccessfully before returning a REDELIVERY_EXHAUSTED error. |  | 
+| Message Digest Algorithm a| String | Secure hashing algorithm to use if the *Use Secure Hash* field is `true`. If the payload of the message is a Java object, Mule ignores this value and returns the value that the payload's `hashCode()` returned. |  | 
+| Message Identifier a| <<RedeliveryPolicyMessageIdentifier>> | One or more expressions that determine if a message was redelivered. This property can be set only if the *Use Secure Hash* field is `false`. |  | 
+| Object Store a| ObjectStore | Configures the object store that stores the redelivery counter for each message. |  | 
+|===
+[[RedeliveryPolicyMessageIdentifier]]
+=== Redelivery Policy Message Identifier
+
+Configures how to identify a redelivered message and how to find out when the message was redelivered.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Use Secure Hash a| Boolean | If `true`, Mule uses a secure hash algorithm to identify a redelivered message. |  | 
+| Id Expression a| String | One or more expressions that determine when a message was redelivered. You can set this property only if the *Use Secure Hash* field is `false`. |  | 
+|===
+
+[[ResourceRequest]]
```

Comment:
do you want to add these new types to the RNs/Upgrade?

Reply:
RNs no. But I'll add them to the Upgrade.